ui/components/multichain/network-list-menu/network-list-menu.tsx

```diff
@@ -485,14 +495,14 @@ export const NetworkListMenu = ({ onClose }: { onClose: () => void }) => {
                   />
                 </Box>
               ) : null}
-
-              {showTestNetworks || currentlyOnTestNetwork ? (
-                <Box className="multichain-network-list-menu">
-                  {searchedTestNetworks.map((network) =>
-                    generateNetworkListItem(network),
-                  )}
-                </Box>
-              ) : null}
+              <Box
+                className="multichain-network-list-menu"
+                hidden={!(showTestNetworks || currentlyOnTestNetwork)}
```

Comment:
Very nice use of hidden here. This way we stabilise the child component, but this parent component can hide/show.

Just want to be potentially wary of doing this pattern everywhere, especially since this means that children are "mounted" and ~useFootGun~ useEffects could still be ran.